diff --git a/include/configs/colibri_imx7_emmc.h b/include/configs/colibri_imx7_emmc.h
index ffd39c3586..1b2687bbd9 100644
--- a/include/configs/colibri_imx7_emmc.h
+++ b/include/configs/colibri_imx7_emmc.h
@@ -141,8 +141,10 @@
 	"boot_file=zImage\0" \
 	"console=ttymxc0\0" \
 	"defargs=\0" \
-	"fdt_board=eval-v3\0" \
+	"fdt_board=rainbow\0" \
 	"fdt_fixup=;\0" \
+   "board=colibri_imx7_rainbow\0" \
+   "board_name=colibri_imx7_rainbow\0" \
 	"m4boot=;\0" \
 	"ip_dyn=yes\0" \
 	"kernel_file=zImage\0" \
@@ -153,16 +155,16 @@
 		"fatload ${interface} 1:1 ${loadaddr} " \
 		"${board}/flash_blk.img && source ${loadaddr}\0" \
 	"setup=setenv setupargs " \
-		"console=tty1 console=${console}" \
+		"console=${console} vt.global_cursor_default=0 " \
 		",${baudrate}n8 ${memargs} consoleblank=0\0" \
 	"setupdate=run setsdupdate || run setusbupdate || run setethupdate\0" \
 	"setusbupdate=usb start && setenv interface usb && " \
 		"fatload ${interface} 0:1 ${loadaddr} " \
 		"${board}/flash_blk.img && source ${loadaddr}\0" \
 	"splashpos=m,m\0" \
-	"videomode=video=ctfb:x:640,y:480,depth:18,pclk:39722,le:48,ri:16,up:33,lo:10,hs:96,vs:2,sync:0,vmode:0\0" \
+	"videomode=video=ctfb:x:800,y:480,depth:18,pclk:39722,le:48,ri:16,up:33,lo:10,hs:96,vs:2,sync:0,vmode:0\0" \
 	"updlevel=2\0" \
-	"vidargs=video=mxsfb:640x480M-16@60"
+	"vidargs=video=mxsfb:800x480M-16@60"
 
 /* Miscellaneous configurable options */
 #define CONFIG_SYS_LONGHELP
